Cargo.toml

```diff
@@ -14,19 +14,21 @@ default-members = ["bin/silius"]
 
 [workspace.package]
 authors = ["Vid Kersic <vid.kersic@yahoo.com>"]
-version = "0.2.0-alpha"
+version = "0.3.0-alpha"
```

Comment:
Next time, We should bump the version first and release.

Reply:
Yeah, that was my bad, I forgot that. Will do it next time

Reply:
Not a big deal right now. Not blaming here. I made the same mistake sometimes. Additionally, we could use bot to release.